### PR TITLE
chore: Add more debugging info for ERTP and objArrayConversions

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
When purses and payments aren't present, report the issuer's alleged name
When objToArray finds a mismatch, report more details

### CODE REVIEW

File: packages/ERTP/src/issuer.js
Comment (katelynsills) [inline packages/ERTP/src/issuer.js]: Does this check make sense? Purse is not an argument here.

File: packages/ERTP/src/issuer.js
Comment (katelynsills) [inline packages/ERTP/src/issuer.js]: I think this could just be:
```suggestion
        fromPaymentsArray.every(assertKnownPayment);
```

**Review by katelynsills — APPROVED** [review]
A few small questions but looks good to me. Let's not merge before Mark approves.

File: packages/ERTP/src/issuer.js
Comment (katelynsills) [reply packages/ERTP/src/issuer.js]: It doesn't hurt to leave it in, but I think it's not checking anything.

Comment (erights) [reply packages/ERTP/src/issuer.js]: I just checked. purses are never deleted from the purseLedger. We don't need assertKnownPurse. Please delete it.

**Review by erights — DISMISSED** [review]
kill assertKnownPurse

**Review by Chris-Hibbert — COMMENTED** [review]
@erights, I couldn't tell if your comment was directed at that specific line, but it sounded more general.

The reason I added this was not for correctness, or to catch deleted purses, it's for debugging, when I accidentally try to do something with a promise that resolves to `undefined` or a non-purse. In those cases, I get no clue as to where the problem occurred. It's extremely frustrating, since I have to do a search over all my new code to find where there's a non-purse. 

With this change, I at least find out which issuer I was trying to do something with, which helps a lot.

File: packages/ERTP/src/issuer.js
Comment (Chris-Hibbert) [reply packages/ERTP/src/issuer.js]: Thanks. I got rid of it.

**Pull request merged**
